chat-widget/src/common/telemetry/defaultConfigs/defaultTelemetryInternalData.ts

```diff
@@ -1,7 +1,7 @@
 import { IInternalTelemetryData } from "../interfaces/IInternalTelemetryData";
 
 export const defaultInternalTelemetryData: IInternalTelemetryData = {
-    environmentVersion: "test",
+    environmentVersion: "prod",
```

Comment:
can we add this change to the change.log?, to communicate the switch from test to prod as default